src/index.html

```diff
@@ -14,11 +14,12 @@
       <h1>what's cookin'?</h1>
       <input class="input" type="text" id ="searchInput" placeholder="Search recipes...">
       <button id ="searchRecipes">search</button>
+      <button class="home-button hidden">home</button>
     </div>
     <div class="nav-div flex-row">
       <button class="hidden">home</button>
```

Comment:
it looks like we already have a home button right here that is just hidden on page load! we should just make sure to delete one of them